helm-charts/nts-pool/templates/cron-config-updater.yaml

```diff
@@ -1,9 +1,26 @@
 
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "nts-pool.fullname" . }}-config-updater
+  namespace: {{ default .Release.Namespace .Values.namespaceOverride }}
+  labels:
+    {{- include "nts-pool.labels" . | nindent 4 }}
+data:
+  config-updater.sh: |
+    #!/bin/sh
+    curl --fail -H "Authentication: ${AUTHTOKEN}" "${CONFIGURL}" |
```

Comment:
Shouldn't this be "Authentication: Bearer ${AUTHTOKEN}"?

Reply:
woeps, fixed